src/sys/unix/net.rs

```diff
@@ -34,12 +34,14 @@ pub fn new_socket(domain: libc::c_int, socket_type: libc::c_int) -> io::Result<l
     let socket = socket.and_then(|socket| {
         // For platforms that don't support flags in socket, we need to
         // set the flags ourselves.
-        syscall!(fcntl(
-            socket,
-            libc::F_SETFL,
-            libc::O_NONBLOCK | libc::O_CLOEXEC
-        ))
-        .map(|_| socket)
+        syscall!(fcntl(socket, libc::F_SETFL, libc::O_NONBLOCK,))
```

Comment:
Weird that rustfmt doesn't remove the last `,`, but no need to change it.

Reply:
rustfmt doesn't generally touch code inside macro invocations.